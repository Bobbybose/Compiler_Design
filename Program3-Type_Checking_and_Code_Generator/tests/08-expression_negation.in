int a = 32;
int b;
b = -a;