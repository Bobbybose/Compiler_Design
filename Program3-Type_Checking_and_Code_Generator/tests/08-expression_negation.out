Code:
%t1 = 32
a = %t1
%t2 = -a

